src/home_robot/home_robot/perception/encoders/__init__.py

```diff
@@ -2,8 +2,9 @@
 #
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
-from typing import Any
+from typing import Any, Union
```

Comment:
why?

Reply:
encode_image can be ndarray or Tensor

Reply:
I agree but i dont see where Union is used in this file though?

Reply:
Oh sure we can remove it